uobtheatre/mail/composer.py

```diff
@@ -0,0 +1,155 @@
+import abc
```

Comment:
This is massively overcomplicating a very simple task. Please don't do this. Just write 2 templates. We won't want the same format in the plain text email anyway.